src/my_cat_gallery.js

```diff
@@ -0,0 +1,84 @@
+(function () {
+    const API_KEY = 'live_wLRK4FXcC44BVnuVXjIPuXo3W3ddMYQlNwySLjQu7z1Vm94HmVnQhf9tC8LaOlxB';
+    const API_URL = 'https://api.thecatapi.com/v1/images';
+    const ITEMS_PER_PAGE = 50;
+
+    // Set up form submission listener for image upload
+    document.getElementById('uploadForm').addEventListener('submit', handleUploadSubmit);
+
+    // Function to handle the image upload process
+    async function handleUploadSubmit(event) {
+        event.preventDefault();
+
+        const fileInput = document.getElementById('fileInput');
```

Comment:
When it's possible it's better to use data directly from form (from submit event). Thus we can have less coupling between DOM and js function. Or, in other words, our function will have less side effects. It is one of design patterns (patterns to write clear and maintainable code),  and we will discuss it more later.